.commitlintrc

```diff
@@ -0,0 +1,9 @@
+{
+  "extends": [
+    "@commitlint/config-conventional",
+  ],
+  "rules": {
+    "body-max-line-length": [2, "always", 72],
+    "header-max-length": [02, "always", 50],
```

Comment:
is there a difference between 2 and 02?

Reply:
Yeah that is odd. Is that in all the instances of this file?

Reply:
No it isn't. Must have been a typo